src/EloquaProvider.php

```diff
@@ -62,7 +63,9 @@ protected function getDefaultHeaders()
      */
     protected function checkResponse(ResponseInterface $response, $data)
```

Comment:
Eloqua will respond with status codes in that diapason if some errors in request params [Validation-errors](http://docs.oracle.com/cloud/latest/marketingcs_gs/OMCAB/index.html#Developers/GettingStarted/API requests/Validation-errors.htm)